portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +28,12 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private static final ImmutableList<String> immutableList = ImmutableList.of("Mango","Strawberry","Banana");
+  Random rand = new Random();
```

Comment:
private Random rand =...
Nothing outside of this class needs to access this variable.